examples/ssh/windows_monitor_iplatency.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Domotz Custom Driver 
+ * Name: Windows Monitor IP Latency
+ * Description: This script is designe to ping an IP address and retrieves the average latency and packet loss percentage.
+ *   
+ * Communication protocol is SSH
+ * 
+ * Tested on Windows Versions:
+ *      - Windows 10
+ *      - Microsoft Windows Server 2019
+ * Powershell Version:
+ *      - 5.1.19041.2364
+ * 
+ * Creates a Custom Driver table with a list of ip addresses , their average latency and their packet loss.
+ * 
+ **/
+
+var pktno = "2"; // Number of packets to send during the ping command.
+var ipAddresses = ["8.8.8.8", "1.1.1.1", "192.168.0.1", "192.168.0.2", "192.168.0.3"]; // List of IP addresses to ping and retrieve status for.
+
+// Set up the SSH command options
+var sshCommandOptions = {
+    "prompt": "]",
+    "timeout": 30000
+};
+
+var tableColumns = D.createTable(
+    "IP Latency",
+    [
+        { label: "IP Address" },
+        { label: "Latency", unit: "ms" },
+        { label: "Packet Loss", unit: "%" }
+    ]
+);
+
+// A function to check for authentication errors during execution
+function checkSshError(err) {
+    if(err.message) console.error(err.message);
+    if(err.code == 5) D.failure(D.errorType.AUTHENTICATION_ERROR);
+    console.error(err);
+    D.failure(D.errorType.GENERIC_ERROR);
+}
+
+function executeCommand(command){
+    var d = D.q.defer();
+    sshCommandOptions.command = command;
+    D.device.sendSSHCommand(sshCommandOptions, function (output, error) {
+        if(error) checkSshError(error);
+        d.resolve(output);
+    });
+    return d.promise;
+}
+
+/**
+ * Excuting a simple command to test access to device:
+ * 'dir' list the files and directories within a specified directory
```

Comment:
This is not a correct validation. Many devices can have "dir" but not the ping command itself with the parameters you need

Please use the ping --help to understand if the device will support any subsequent request